web-components/src/components/advance-list/AdvanceList.ts

```diff
@@ -12,13 +12,18 @@ export namespace AdvanceList {
   export class ELEMENT extends FocusMixin(LitElement) {
     @property({ type: Array }) items: any[] = [];
     @property({ type: Boolean }) isLoading = false;
-    @property({ type: String }) selectedItemId = "";
-    @property({ type: String }) value = "";
+    @property({ type: Boolean }) isMultiSelectEnabled = false;
```

Comment:
Combobx web component has attribute called isMulti can we name same here?
@property({ type: Boolean, attribute: "is-multi" }) isMulti = false;